python/tests/mlir/qreg_iterable.py

```diff
@@ -43,6 +43,6 @@ def foo(N: int):
 # CHECK:           ^bb0(%[[VAL_12:.*]]: i64):
 # CHECK:             %[[VAL_13:.*]] = arith.addi %[[VAL_12]], %[[VAL_1]] : i64
 # CHECK:             cc.continue %[[VAL_13]] : i64
-# CHECK:           } {invariant}
+# CHECK:           }
```

Comment:
This loop is invariant as it will iterate once on each qubit in the qvector. Did you mean to omit the attribute?

On the other hand, loop analysis will detect that it is an invariant loop, so there's no correctness issue.